packages/manager/modules/vcd-api/src/hook/useDeleteVcdVrackSegment.ts

```diff
@@ -0,0 +1,46 @@
+import { ApiError } from '@ovh-ux/manager-core-api';
+import {
+  useMutation,
+  UseMutationOptions,
+  useQueryClient,
+} from '@tanstack/react-query';
+import { deleteVrackSegment } from '../api/vcd-vrack-network';
+import {
+  deleteVdcVrackSegmentMutationKey,
+  getVcdVrackNetworkQueryKey,
+} from '../utils';
+
+export const useDeleteVcdVrackSegment = ({
+  id,
+  vcdId,
+  vrackSegmentId,
+  onSuccess,
+  ...options
+}: {
+  id: string;
+  vcdId: string;
+  vrackSegmentId: string;
+} & Partial<UseMutationOptions<unknown, ApiError, void, unknown>>) => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationKey: deleteVdcVrackSegmentMutationKey({
+      id,
+      vcdId,
+      vrackSegmentId,
+    }),
```

Comment:
Not that it harms anything, but I still don't get why everybody uses mutation keys in the manager code ?
I never used them.
It looks like a lot of code to me and I really do not see what purpose it serves in the manager code.

About mutation keys : https://github.com/TanStack/query/discussions/5787